docs/integrations/README.md

```diff
@@ -12,22 +12,10 @@ Airbyte uses a grading system for connectors to help you understand what to expe
 
 For more information about the grading system, see [Product Release Stages](https://docs.airbyte.com/project-overview/product-release-stages)
 
-## Sources
-
-<iframe
-    src="https://airbyte.metabaseapp.com/public/question/9af21fd5-ff87-4809-8e04-966207490e0b#titled=false&hide_download_button=true"
-    frameborder="0"
-    width="600"
-    height="2000"
-    allowtransparency
-></iframe>
-
-## Destinations
-
 <iframe
-    src="https://airbyte.metabaseapp.com/public/question/263d737e-533a-4cc2-a1cf-4c8c7eca87d6#titled=false&hide_download_button=true"
+    src="https://connectors.airbyte.com/files/generated_reports/connector_registry_report.html"
```

Comment:
My feedback is mostly on this page:
https://connectors.airbyte.com/files/generated_reports/connector_registry_report.html

- Overall, super happy to be replacing the Metabase chart with another table! 

Some things we have in the old version that would be nice in the new version:
- The connector name is clickable 
- There is a search bar (idk if people use this... I do sometimes)

Some things in the new table that I'm not sure are necessary:
- Definition Id
- Icon
- Connector Type (it is in the table title)

Reply:
Over on the far end there are a bunch more links now.  I think this is an improvement, although maybe not as discoverable.  
<img width="899" alt="Screenshot 2023-04-27 at 2 57 20 PM" src="https://user-images.githubusercontent.com/303226/234999883-071560eb-f941-4241-aed4-30c6ae6ebd3f.png">

Reply:
It's not so much that those 3 things aren't needed, but maybe should be moved to the end of the list?

Reply:
Oops, we commented at the same time 😄 

The "3 things" I am referring to are: Id, Icon, Connector Type


The 3 links are great! I think those should stay... AND can we also make the Connector Name clickable?

Reply:
Thanks for the comments alex!

So this report is serving a few purposes atm
1. Replacing our list of which builds are often passing or failing
2. Replacing which connectors are deployed and where
3. Replacing the metabase table to show people which connectors are available in which platform (and what release stage)

So theres a few extra fields for sure but I think they are useful for the other use cases 😅 

Also Im thinking command+f could be a good enough search for now.

LMK if you disagree with either of those!

In terms of a clickable connector name we can add that!

Reply:
Thanks Ben! 
I was also thinking `command+f` is probably what most people use, so I'm good with that! 

Overall, I think all of the columns are useful, but it would be great to prioritize the ones that are more meaningful to be farther to the left. Is that a change you are open to, or does that impact some of the other use cases?

Reply:
That seems reasonable to me!

I know TCS uses the catalog frequently, so please make sure to communicate the change with them, too 🙏

Reply:
@evantahler where would the best place to leave feedback be?

Reply:
Add feedback here! https://github.com/airbytehq/airbyte/issues/25678

Reply:
@TBernstein4 @alex-gron Ive got an issue here: [Connector Report Improvements](https://app.zenhub.com/workspaces/connector-operations-6318ede5eda32e0010ef8566/issues/gh/airbytehq/airbyte/25679) for us to continue the discussion

Reply:
Race condition! use https://github.com/airbytehq/airbyte/issues/25678 please :D